apps/web/src/components/docs/Docs.tsx

```diff
@@ -0,0 +1,333 @@
+  }, [code]);
+
+  // Workaround for img tags that is not parsed correctly by mdx-bundler
+  useEffect(() => {
+    if (Component === null || isLoading) {
+      return;
+    }
+
+    const docs = document.getElementById('docs');
+
+    if (!docs) {
+      return;
+    }
+
+    const images = Array.from(docs.getElementsByTagName('img'));
+
+    for (const img of images) {
+      if (img.src.startsWith(MINTLIFY_IMAGE_URL)) {
+        continue;
+      }
+
+      const url = new URL(img.src);
+      img.src = `${MINTLIFY_IMAGE_URL}${url.pathname}`;
+    }
+  }, [Component, isLoading]);
+
+  if (isLoading) {
+    return (
+      <Center
+        className={css({
+          marginTop: '[4rem]',
+        })}
+      >
+        <Loader color={colors.error} size={32} />
+      </Center>
+    );
+  }
+
+  if (Component === null) {
+    return (
+      <Flex
+        className={css({
+          marginTop: '[4rem]',
+        })}
+        justify="center"
+        align="center"
+      >
+        <Text>We could not load the documentation for you. Please try again.</Text>
+        {children}
+      </Flex>
+    );
+  }
+
+  return (
+    <>
+      <VStack
+        alignItems="unset"
+        id="docs"
+        gap={12}
+        className={css({
+          textAlign: 'justify left',
+        })}
+      >
+        <div>
+          <Flex justify="space-between" align="center">
+            <TitleH1>{title}</TitleH1>
+            {actions}
+          </Flex>
+          <Text>{description}</Text>
+        </div>
+        <Component
+          components={{
+            tr: ({ className, ...props }: any) => {
+              return (
+                <tr
+                  {...props}
+                  className={css({
+                    height: '250',
+                    lineHeight: '125',
+                    borderBottom: 'solid',
+                    borderBottomColor: 'legacy.B20',
+                  })}
+                />
+              );
+            },
+            thead: ({ className, ...props }: any) => {
+              return (
+                <thead
+                  {...props}
+                  className={css({
+                    height: '150',
+                    lineHeight: '125',
+                    color: 'typography.text.main',
+                    borderBottomColor: 'legacy.B30',
+                  })}
+                />
+              );
+            },
+            Frame: ({ className, ...props }: any) => {
+              return (
+                <div {...props}>
+                  <img alt="" src={`${MINTLIFY_IMAGE_URL}${props.children.props.src}`} />
+                  <Text className={css({ textAlign: 'center', fontStyle: 'italic' })}>{props.caption}</Text>
+                </div>
+              );
+            },
+            Info: ({ className, ...props }: any) => {
+              return (
+                <Alert style={{ borderRadius: 12 }} color="gray" {...props} icon={<IconInfoOutline color="white" />} />
```

Comment:
```suggestion
                <Alert className={css({ borderRadius: '150' })}  {...props} icon={<IconInfoOutline />} />
```
Please add "color" a different way